src/planet-a/Active/SysPlanetA.cpp

```diff
@@ -75,30 +74,34 @@ StatusActivated SysPlanetA::activate_sat(
     ActiveEnt root = scene.hier_get_root();
     ActiveEnt planetEnt = scene.hier_create_child(root, "Planet");
 
-    auto &planetTransform = scene.get_registry()
-                            .emplace<osp::active::ACompTransform>(planetEnt);
-    planetTransform.m_transform = Matrix4::translation(positionInScene);
+    auto &rPlanetTransform = scene.get_registry()
+                             .emplace<osp::active::ACompTransform>(planetEnt);
```

Comment:
Are you following an 80 column rule or something?

I'd argue that if you are, 120 columns might be better. Basically no one has screens that small anymore, and if they do, they're on their own.

Reply:
With a split window, my laptop only manages 83 columns wide :'(

Reply:
O_o?

My laptop can do like 150 columns.

Where's the mismatch?

Reply:
It's a small (14") 1600x900 display, and Ubuntu cleverly adjusts the text scaling to make the font very readable. I could shrink the scale but it would make it a lot harder to read.

Wait, 150? On both sides of a split screen? My full size 1920x1080 desktop monitor only gives me 103, 117 if I use the default zoom (although VS does have some clutter around the edges of the window). You say your eyesight sucks but I think your code is just half of a comfortably readable size haha

Reply:
Yes, I am following an 80 column rule. The verbosity of modern C++ code makes this a bit difficult to follow. I usually code with a window vertically split on half of a 1080p screen, and sometimes use a second monitor. Personally, I find it easier to read text that extends more vertically than horizontally.

Reply:
What is a "split screen" then? I just use my whole screen for my editor.

Reply:
Having two files open side by side on the same monitor?